groovy/postInit/components/IntegratedCircuits.groovy

```diff
@@ -1,12 +1,244 @@
+def ASSEMBLER = recipemap('assembler')
+def VACUUM = recipemap('vacuum_chamber')
+def CIRCUIT_ASSEMBLER = recipemap('circuit_assembler')
+
 //Change transistor to use silicon wafer instead of silicon plate
 // Transistor * 8
 mods.gregtech.assembler.removeByInput(120, [metaitem('plateSilicon'), metaitem('wireFineTin') * 6], [fluid('plastic') * 144])
 
-mods.gregtech.assembler.recipeBuilder()
+ASSEMBLER.recipeBuilder()
         .inputs(ore('wireFineTin') * 6)
         .inputs(metaitem('wafer.silicon'))
         .fluidInputs(fluid('plastic') * 144)
         .outputs(metaitem('component.transistor') * 8)
         .duration(160)
         .EUt(120)
-        .buildAndRegister()
\ No newline at end of file
+        .buildAndRegister()
+
+CIRCUIT_ASSEMBLER.recipeBuilder()
+        .inputs(ore('wireFineGold') * 6)
+        .inputs(ore('componentCapacitor'))
+        .inputs(ore('componentResistor') * 2)
+        .inputs(ore('componentTransistor') * 4)
+        .inputs(ore('componentDiode') * 2)
+        .inputs(metaitem('board.phenolic'))
+        .fluidInputs(fluid('soldering_alloy') * 72)
+        .outputs(metaitem('op_amp'))
+        .duration(100)
+        .EUt(30)
+        .buildAndRegister()
+
+CIRCUIT_ASSEMBLER.recipeBuilder()
+        .inputs(ore('wireFineGold') * 6)
+        .inputs(ore('componentCapacitor'))
+        .inputs(ore('componentResistor') * 2)
+        .inputs(ore('componentTransistor') * 4)
+        .inputs(ore('componentDiode') * 2)
+        .inputs(metaitem('board.phenolic'))
+        .fluidInputs(fluid('tin') * 144)
+        .outputs(metaitem('op_amp'))
+        .duration(100)
+        .EUt(30)
+        .buildAndRegister()
+
+CIRCUIT_ASSEMBLER.recipeBuilder()
+        .inputs(ore('wireFineGold') * 3)
+        .inputs(ore('componentCapacitor'))
+        .inputs(ore('componentResistor') * 2)
+        .inputs(ore('componentTransistor') * 4)
+        .inputs(ore('componentDiode') * 2)
+        .fluidInputs(fluid('silicone_rubber') * 72)
```

Comment:
typically epoxy cresol novolacs plastic is used as a casing for potted components, i can take care of the recipe for that.